File: chat-service/src/main/java/com/relay/chatservice/chat/ChatNotification.java
package com.relay.chatservice.chat;

import lombok.*;
import org.springframework.data.mongodb.core.mapping.Document;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Document
public class ChatNotification {
    private String id;
    private String senderId;
    private String recipientId;
    private String content;
}

File: chat-service/src/main/java/com/relay/chatservice/chat/ChatNotification.kt
package com.relay.chatservice.chat

import org.springframework.data.mongodb.core.mapping.Document

@Document
data class ChatNotification(
    var id: String? = null,
    var senderId: String? = null,
    var recipientId: String? = null,
    var content: String? = null
)
